test_runner/fixtures/workload.py

```diff
@@ -21,12 +21,23 @@ class Workload:
     - reads, checking we get the right data (`validate`)
     """
 
-    def __init__(self, env: NeonEnv, tenant_id: TenantId, timeline_id: TimelineId):
+    def __init__(
+        self,
+        env: NeonEnv,
+        tenant_id: TenantId,
+        timeline_id: TimelineId,
+        branch_name: Optional[str] = None,
+    ):
         self.env = env
         self.tenant_id = tenant_id
         self.timeline_id = timeline_id
         self.table = "foo"
 
+        if branch_name is None:
```

Comment:
```suggestion
        self.branch_name = branch_name or "main"
```

Reply:
I don't mind either way - applied by hand in 4c1d6d08d